src/norms/Norms_Base.cc

```diff
@@ -15,14 +15,9 @@ using namespace rtt_norms;
 
 //----------------------------------------------------------------------------//
 //! Default constructor.
-Norms_Base::Norms_Base(void)
+Norms_Base::Norms_Base()
     : d_sum_L1(-42.0), d_sum_L2(-42.0), d_Linf(-42.0), d_sum_weights(-42.0) {
-  reset();
-}
-
-//----------------------------------------------------------------------------//
-//! Destructor.
-Norms_Base::~Norms_Base() { /* empty */
+  Norms_Base::reset();
```

Comment:
I'm curious about this -- why is this reset() call prefixed with the class?

Reply:
That is an old code style.  Circa the 98 standard, there was a convention to use the base class name to scope calls like this.  Another one that is used in some codes is `this->reset()`.  It also ensures that you are not calling a `reset` function from a derived class.